nvdaHelper/vbufBackends/mshtml/node.cpp

```diff
@@ -353,11 +353,17 @@ MshtmlVBufStorage_controlFieldNode_t::~MshtmlVBufStorage_controlFieldNode_t() {
 }
 
 void MshtmlVBufStorage_controlFieldNode_t::preProcessLiveRegion(const MshtmlVBufStorage_controlFieldNode_t* parent, const std::map<std::wstring,std::wstring>& attribsMap) {
-	 auto i=attribsMap.find(L"HTMLAttrib::aria-live");
+	auto i=attribsMap.find(L"HTMLAttrib::aria-live");
 	if(i!=attribsMap.end()&&!i->second.empty()) {
 		this->ariaLiveNode=((i->second.compare(L"polite")==0)||(i->second.compare(L"assertive")==0))?this:NULL;
+		this->ariaLivePoliteness = i->second;
 	} else {
 		this->ariaLiveNode=parent?parent->ariaLiveNode:NULL;
+		if (this->ariaLiveNode) {
+			this->ariaLivePoliteness = static_cast<MshtmlVBufStorage_controlFieldNode_t*>(this->ariaLiveNode)->ariaLivePoliteness;

```

Comment:
Re: change the type of `ariaLiveNode` this cast would not be necessary. While you are at it, could you look at the other usages of ariaLiveNode and clean up those lines (remove cast, use nullptr, fix formatting). Despite making the diff longer, I think it will improve readability of this PR because it will give more context in the diff.